src/utils/ast-helpers.ts

```diff
@@ -179,22 +183,23 @@ export function findPropertyCallExpressions(node: Node, prop: string, fnName: st
 	if (Array.isArray(fnName)) {
 		fnName = fnName.join('|');
 	}
-	const query = 'CallExpression > ' +
-        `PropertyAccessExpression:has(Identifier[name=/^(${fnName})$/]):has(PropertyAccessExpression:has(Identifier[name="${prop}"]):has(ThisKeyword)) > ` +
-        `PropertyAccessExpression:has(ThisKeyword) > Identifier[name="${prop}"]`;
-	const nodes = tsquery<Identifier>(node, query);
-	// Since the direct descendant operator (>) is not supported in :has statements, we need to
-	// check manually whether everything is correctly matched
-	const filtered = nodes.reduce<CallExpression[]>((result: CallExpression[], n: Node) => {
-		if (
-			tsquery(n.parent, 'PropertyAccessExpression > ThisKeyword').length > 0 &&
-			tsquery(n.parent.parent, `PropertyAccessExpression > Identifier[name=/^(${fnName})$/]`).length > 0
-		) {
-			result.push(n.parent.parent.parent as CallExpression);
+
+	const query = `CallExpression > PropertyAccessExpression:has(Identifier[name=/^(${fnName})$/]):has(PropertyAccessExpression:has(ThisKeyword))`;
+	const result = tsquery<PropertyAccessExpression>(node, query);
+
+	const nodes: CallExpression[] = [];
+	result.forEach((n) => {
```

Comment:
Similar here, I reduced the specificity of the query and then manually filter out the `CallExpression` nodes from the AST, so that both private and public properties are matched.